src/ContainerCommitModal.jsx

```diff
@@ -67,119 +50,116 @@ class ContainerCommitModal extends React.Component {
-                               onClick={Dialogs.close}>
-                           {_("Cancel")}
-                       </Button>
-                   </>}
-            >
-                {commitContent}
-            </Modal>
-        );
-    }
-}
+    };
+
+    const commitContent =
+        <Form isHorizontal>
+            <FormGroup fieldId="commit-dialog-image-name" label={_("New image name")}
+                       validated={nameError ? "error" : "default"}
+                       helperTextInvalid={nameError}>
+                <TextInput id="commit-dialog-image-name"
+                           value={imageName}
+                           validated={nameError ? "error" : "default"}
+                           onChange={value => { setNameError(""); setImageName(value) }} />
+            </FormGroup>
+
+            <FormGroup fieldId="commit-dialog-image-tag" label={_("Tag")}>
+                <TextInput id="commit-dialog-image-tag"
+                           placeholder="latest" // Do not translate
+                           value={tag}
+                           onChange={value => { setNameError(""); setTag(value) }} />
+            </FormGroup>
+
+            <FormGroup fieldId="commit-dialog-author" label={_("Author")}>
+                <TextInput id="commit-dialog-author"
+                           placeholder={_("Example, Your Name <yourname@example.com>")}
+                           value={author}
+                           onChange={setAuthor} />
+            </FormGroup>
+
+            <FormGroup fieldId="commit-dialog-command" label={_("Command")}>
+                <TextInput id="commit-dialog-command"
+                           value={command}
+                           onChange={setCommand} />
+            </FormGroup>
+
+            <FormGroup fieldId="commit-dialog-pause" label={_("Options")} isStack hasNoPaddingTop>
+                <Checkbox id="commit-dialog-pause"
+                          isChecked={pause}
+                          onChange={setPause}
+                          label={_("Pause container when creating image")} />
+                <Checkbox id="commit-dialog-docker"
+                          isChecked={useDocker}
+                          onChange={setUseDocker}
+                          description={_("Docker format is useful when sharing the image with Docker or Moby Engine")}
+                          label={_("Use legacy Docker format")} />
+            </FormGroup>
+        </Form>;
+
+    return (
+        <Modal isOpen
+                 showClose={false}
+                 position="top" variant="medium"
+                 title={_("Commit container")}
+                 description={fmt_to_fragments(_("Create a new image based on the current state of the $0 container."), <b>{container.Names}</b>)}
+                 footer={<>
+                     {dialogError && <ErrorNotification errorMessage={dialogError} errorDetail={dialogErrorDetail} onDismiss={() => setDialogError("")} />}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1146-20221125-083242-f4438564-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)